_posts/2023-04-04-flux-accounting-0.23.0.md

```diff
@@ -0,0 +1,21 @@
+---
+title: flux-accounting v0.23.0
+date: 2023-04-04 00:00:00
+author: "flux-framework"
+categories: 'release'
+version: 0.23.0
+download: https://github.com/flux-framework/flux-accounting/releases/download/v0.23.0/flux-accounting-0.23.0.tar.gz
+---
+
+Download from GitHub [here]({{ page.download }})
+
+# Release Notes
+
+flux-accounting version 0.23.0 - 2023-04-04
+-------------------------------------------
+
+#### Fixes
+
+* bindings: raise error to caller (#327, #328, #329)
```

Comment:
We likely want the listing here to be populated with links - @cmoussa1 is this a format you use regularly, so A) should we try to parse it here with the action, or B) be sure to provide one issue per line instead, or C) flag it as rare and manually fix here, or D) something else?

Reply:
Ah, so sorry @vsoch! I didn't know that using one more than one issue per line would remove the link functionality with these. 🤦 This should be a one time thing. Every one of those numbered PRs targeted the same fix, so that's why I included them on the same line.

I'd be happy to fix manually if that is okay. Should I just pull this branch down, edit the commit, and force push back up? I'm not sure how this repository is structured and I don't want to mess anything up!

Reply:
FWIW, I sometimes also combine multiple PRs on a single line. It would be cool if the action supported it.